src/Features/Core/Portable/CodeFixes/Configuration/ConfigureSeverity/ConfigureSeverityLevelCodeFixProvider.cs

```diff
@@ -58,11 +58,11 @@ private static ImmutableArray<CodeFix> GetConfigurations(Project project, IEnume
             foreach (var diagnostic in diagnostics)
             {
                 var nestedActions = ArrayBuilder<CodeAction>.GetInstance();
-                foreach (var (name, value) in s_editorConfigSeverityStrings)
+                foreach (var (name, value, title) in s_editorConfigSeverityStrings)
                 {
                     nestedActions.Add(
                         new SolutionChangeAction(
-                            name,
+                            title,
                             solution => ConfigurationUpdater.ConfigureSeverityAsync(value, diagnostic, project, cancellationToken),
                             name));
```

Comment:
Is name only being used here for the equivalence key? I'd imagine we can just use the value then?

Reply:
Are you saying we can get rid of the 'name' here?
Yes that possible

Reply:
Resolved